README.md

```diff
@@ -419,7 +419,7 @@ check_suite.load_all_available_checkers()
 # Run cf and adcc checks with normal strictness, verbose text format to stdout
 return_value, errors = ComplianceChecker.run_checker(
                                 '/path/or/url/to/your/dataset',
-                                ['cf', 'acdd'], 0, 'normal', '-', 'text')
+                                ['cf', 'acdd'], 0, 'normal')
```

Comment:
Good catch! We're going to be updating the documentation as part of our next milestone. I like the idea of showing all the arguments in the example, so can you change this line to ```['cf', 'acdd'], 0, 'normal', None, '-', 'text')```?